worldengine/generation.py

```diff
@@ -142,56 +150,55 @@ def harmonize_ocean(ocean, elevation, ocean_level):
 
 def sea_depth(world, sea_level):
 
-    # a dynamic programming approach to gather how far the next land is 
+    # a dynamic programming approach to gather how far the next land is
     # from a given coordinate up to a maximum distance of max_radius
     # result is 0 for land coordinates and -1 for coordinates further than
     # max_radius away from land
     # there might be even faster ways but it does the trick
 
     def next_land_dynamic(ocean, max_radius=5):
-    
         next_land = numpy.full(ocean.shape, -1, int)
 
         # non ocean tiles are zero distance away from next land
-        next_land[numpy.logical_not(ocean)]=0
+        next_land[numpy.logical_not(ocean)] = 0
 
         height, width = ocean.shape
 
         for dist in range(max_radius):
-            indices = numpy.transpose(numpy.where(next_land==dist))
+            indices = numpy.transpose(numpy.where(next_land == dist))
             for y, x in indices:
                 for dy in range(-1, 2):
                     ny = y + dy
                     if 0 <= ny < height:
                         for dx in range(-1, 2):
                             nx = x + dx
                             if 0 <= nx < width:
-                                if next_land[ny,nx] == -1:
-                                    next_land[ny,nx] = dist + 1
+                                if next_land[ny, nx] == -1:
+                                    next_land[ny, nx] = dist + 1
         return next_land
 
-    # We want to multiply the raw sea_depth by one of these factors 
+    # We want to multiply the raw sea_depth by one of these factors
     # depending on the distance from the next land
     # possible TODO: make this a parameter
     factors = [0.0, 0.3, 0.5, 0.7, 0.9]
 
     next_land = next_land_dynamic(world.layers['ocean'].data)
 
-    sea_depth = sea_level - world.layers['elevation'].data
+    result = sea_level - world.layers['elevation'].data
```

Comment:
I do not like the name result much here

Reply:
sea_depth is the name of the function and pylint rightfully complains that this redefines a name from the outer scope. Personally I use `result` a lot as a variable name, but I'm open to anything that is not `sea_depth`. Alternatively the function needs a new name.

Reply:
depth_of_sea  (in the very french way of saying sea_depth)